src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -68,6 +71,8 @@ object ESType extends ESPropertyFields {
 
 }
 
+case class ESNestedType(properties:Map[String,ESInnerField], `type`:String="nested") extends ESPropertyFields
```

Comment:
The `ESNestedType` is only used to create a mapping in ES - it's not used to hold values coming from the ontology. So I don't think this comment is applicable?